api/app/signals/apps/reporting/csv/datawarehouse.py

```diff
@@ -242,6 +269,10 @@ def _create_category_assignments_csv(location):
             ) for c in Category.objects.prefetch_related('departments').all()
         }
 
+        slo_cache = {}
+        for c in Category.objects.prefetch_related('departments').all():
+            slo_cache[c.id] = c.slo.order_by('-created_at').first() if c.slo.exists() else None
```

Comment:
Only having the most recent SLO is consistent with the other fields on the category model that may get changed, but our users are probably also interested in the history of the SLO entries. Since we have the SLO history, I would propose to also generate a `slo.csv` with all SLO entries over time (with values, timestamp and reference to a category). That way our users will not need to reconstruct the relevant history.